src/modules/components/proposal/proposalDataListItem/proposalDataListItemStructure/proposalDataListItemStructure.tsx

```diff
@@ -32,36 +32,61 @@ export const ProposalDataListItemStructure: React.FC<IProposalDataListItemStruct
                 <p className="line-clamp-2 leading-normal text-neutral-500 md:text-lg">{summary}</p>
             </div>
 
-            {ongoing && type === 'approvalThreshold' && <ApprovalThresholdResult {...result} />}
+            {ongoing && type === 'approvalThreshold' && result && <ApprovalThresholdResult {...result} />}
 
-            {ongoing && type === 'majorityVoting' && <MajorityVotingResult {...result} />}
+            {ongoing && type === 'majorityVoting' && result && <MajorityVotingResult {...result} />}
 
-            <div className="flex items-center gap-x-4 md:gap-x-6">
-                <div className="flex min-h-5 flex-1 items-center gap-x-0.5 text-sm leading-tight text-neutral-600 md:min-h-6 md:text-base">
-                    {/* TODO: apply internationalization [APP-2627] */}
+            <div className="flex items-center justify-between gap-x-4 md:gap-x-6">
+                <div
+                    className={classNames(
+                        'inline-grid auto-cols-auto grid-flow-col content-center',
+                        'min-h-5 gap-x-0.5 text-sm leading-tight text-neutral-600 md:min-h-6 md:gap-x-1 md:text-base',
+                    )}
+                >
                     By
-                    {/* using solution from https://kizu.dev/nested-links/ to nest anchor tags */}
-                    <object type="disregardType" className="ml-0.5 md:ml-1">
-                        <Link href={publisherProfileLink}>{publisherLabel}</Link>
-                    </object>
+                    {showParsedPublisher === false && <span>3+ creators</span>}
+                    {showParsedPublisher &&
+                        parsedPublisher.map(({ label, link }, index) => (
+                            <span key={label} className="truncate">
+                                <object type="disregardType" className="flex shrink">
+                                    {link != null && (
+                                        // using solution from https://kizu.dev/nested-links/ to nest anchor tags
+                                        <Link href={link}>{label}</Link>
+                                    )}
+                                    {link == null && <span className="truncate">{label}</span>}
+                                    {index < parsedPublisher.length - 1 && ','}
+                                </object>
+                            </span>
+                        ))}
                 </div>
-
-                {/* TODO: apply internationalization [APP-2627] */}
-                {protocolUpdate && <Tag label="OSx Update" variant="primary" />}
+                {tag && <Tag label={tag} variant="primary" />}
             </div>
         </DataList.Item>
     );
 };
+
+function parsePublisher(publisher: IPublisher, isConnected: boolean, connectedAddress: string | undefined) {
+    const publisherIsConnected = isConnected && connectedAddress?.toLowerCase() === publisher.address.toLowerCase();
```

Comment:
I'd suggest to always use the `isAddressEqual` functionality from viem to compare addresses

Reply:
In this case it overcomplicates things for no real benefit considering it does the same underlying calculation and more importantly, throws if either property isn't a valid `Address`. Both `address` and `connectedAddress` can be invalid addresses

Reply:
Oh I see, I think it would be great to have such an utility on `addressUtils` that we can reuse every time we need to compare addresses so that we don't forget to lower case it. It can be simply as:
```
const isAddressEqual = (addressOne?: string, addressTwo?: string) => this.isAddress(addressOne) && this.isAddress(addressTwo) && isAddressEqual(addressOne, addressTwo)
```
But I can create a ticket for it and do it on a separate PR